app/views/layouts/_nav.html.erb

```diff
@@ -0,0 +1,32 @@
+<nav class="navbar navbar-inverse">
+  <div class="container-fluid">
+    <div class="navbar-header">
+      <ul class="nav nav-pills">
+        <li role="presentation"><%= link_to "Home", root_path, style: "color:turquoise" %></li>
+        <li role="presentation"> <%= link_to "About", about_path, style: "color:turquoise" %></li>
+        <li role="presentation"> <%= link_to "Show All Posts", posts_path, style: "color:turquoise" %></li>
+        <li role="presentation"> <%= link_to "Create New Post", new_post_path, style: "color:turquoise" %></li>
```

Comment:
Any reason you chose to put these in with style, rather than using a css class/id? (Once again, I'm just throwing these types of comments in because your code looks excellent to me and I want to try and contribute a few things at least haha)